tests/test_art_packs.py

```
@@ -173,3 +173,56 @@ async def fail_refresh(*_args: object) -> None:
+    candidate = types.SimpleNamespace(
+        item_id="mlg7qm",
+        image_url="https://w.wallhaven.cc/full/ml/wallhaven-mlg7qm.jpg",
+        thumb_url="https://th.wallhaven.cc/lg/ml/mlg7qm.jpg",
+        title="General Wallpaper mlg7qm",
+        artist=None,
+        license=None,
+        attribution="General Wallpaper mlg7qm, Wallhaven",
+        extra={"source_url": "https://wallhaven.cc/w/mlg7qm"},
+    )
+
+    async def browse(_hass, actual_entry, provider_key, provider_path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `browse`

(ANN202)